### PR TITLE
Config read workaround

### PR DESCRIPTION
**Pull request by volodymyrss**
not at all convinced this the way, but the problem is that config is read from file in dispatcher, need to investigate
for now this works

### CODE REVIEW

File: dispatcher_plugin_nb2workflow/dataserver_dispatcher.py [hunk 1]
```diff
@@ -2,11 +2,17 @@
...
 
 class NB2WDataDispatcher:
     def __init__(self, instrument=None, param_dict=None, task=None, config=None):
         if config is None:
-            config = DataServerConf.from_conf_dict(instrument.data_server_conf_dict)
+            try:
+                config = DataServerConf.from_conf_dict(instrument.data_server_conf_dict)
+            except:
+                #this happens if the instrument is not found in the instrument config, which is always read from a static file
+                config = DataServerConf.from_conf_dict(exposer.read_conf_file()['instruments'][instrument.name])
```
Comment (dsavchenko) [inline dispatcher_plugin_nb2workflow/dataserver_dispatcher.py:14]: Doesn't it always produce an exception? `exposer.read_conf_file()['instruments']` is empty dict if nothing is in KG so second getitem will fail

**Review by dsavchenko — COMMENTED** [review]
Not sure, I fully understand the reasoning below this

**Comment by volodymyrss** [comment]
> Not sure, I fully understand these changes. Seems that the plugin works without them. Could you point out what does it fix?

does it work for you for dynamically configured instruments? those which are not in the file?

**Review by dsavchenko — APPROVED** [review]

**Comment by volodymyrss** [comment]
I'd need to test it though

**Pull request merged**